### PR TITLE
CF-398 Improving waiting VM accessibility by SSH in generate load.

### PR DESCRIPTION
**Pull request by plomakin**
Fixed VM accessability by floating IP in generate_load.py

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check failed
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/3189/

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferry_devlab.tests.utils
cloudferry_devlab/cloudferry_devlab/tests/utils.py:16: [W0611(unused-import), ] Unused 
import time

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by SVilgelm** [comment]
:+1:

**Comment by mgrygoriev** [comment]
+1, works in devlab

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferry_devlab.tests.utils
cloudferry_devlab/cloudferry_devlab/tests/utils.py:17: [E0401(import-error), ] Unable 
to import 'fabric'
cloudferry_devlab/cloudferry_devlab/tests/utils.py:18: [E0401(import-error), ] Unable 
to import 'fabric'
cloudferry_devlab/cloudferry_devlab/tests/utils.py:19: [E0401(import-error), ] Unable 
to import 'neutronclient.common'
cloudferry_devlab/cloudferry_devlab/tests/utils.py:20: [E0401(import-error), ] Unable 
to import 'yaml'
************* Module cloudferry_devlab.tests.testcases.test_resource_migration
cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py:20: 
[E0401(import-error), ] Unable to import 'fabric.api'
cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py:21: 
[E0401(import-error), ] Unable to import 'fabric.network'
cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py:22: 
[E0401(import-error), ] Unable to import 'nose.plugins.attrib'
************* Module cloudferry_devlab.generate_load
cloudferry_devlab/cloudferry_devlab/generate_load.py:22: [E0401(import-error), ] Unable 
to import 'keystoneclient'
cloudferry_devlab/cloudferry_devlab/generate_load.py:23: [E0401(import-error), ] Unable 
to import 'neutronclient.common'
cloudferry_devlab/cloudferry_devlab/generate_load.py:24: [E0401(import-error), ] Unable 
to import 'novaclient'
cloudferry_devlab/cloudferry_devlab/generate_load.py:25: [E0401(import-error), ] Unable 
to import 'yaml'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by plomakin** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferry_devlab.tests.utils
cloudferry_devlab/cloudferry_devlab/tests/utils.py:17: [E0401(import-error), ] Unable 
to import 'fabric'
cloudferry_devlab/cloudferry_devlab/tests/utils.py:18: [E0401(import-error), ] Unable 
to import 'fabric'
cloudferry_devlab/cloudferry_devlab/tests/utils.py:19: [E0401(import-error), ] Unable 
to import 'neutronclient.common'
cloudferry_devlab/cloudferry_devlab/tests/utils.py:20: [E0401(import-error), ] Unable 
to import 'yaml'
************* Module cloudferry_devlab.tests.testcases.test_resource_migration
cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py:20: 
[E0401(import-error), ] Unable to import 'fabric.api'
cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py:21: 
[E0401(import-error), ] Unable to import 'fabric.network'
cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py:22: 
[E0401(import-error), ] Unable to import 'nose.plugins.attrib'
************* Module cloudferry_devlab.generate_load
cloudferry_devlab/cloudferry_devlab/generate_load.py:22: [E0401(import-error), ] Unable 
to import 'keystoneclient'
cloudferry_devlab/cloudferry_devlab/generate_load.py:23: [E0401(import-error), ] Unable 
to import 'neutronclient.common'
cloudferry_devlab/cloudferry_devlab/generate_load.py:24: [E0401(import-error), ] Unable 
to import 'novaclient'
cloudferry_devlab/cloudferry_devlab/generate_load.py:25: [E0401(import-error), ] Unable 
to import 'yaml'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by mgrygoriev** [comment]
+1

File: cloudferry_devlab/cloudferry_devlab/generate_load.py [hunk 1]
```diff
@@ -637,7 +637,10 @@ def wait_until_vms_with_fip_accessible(_vm_id):
                 fip_addr = self.migration_utils.get_vm_fip(vm)
             except RuntimeError:
                 return
-            self.migration_utils.wait_until_vm_accessible_via_ssh(fip_addr)
+            base.BasePrerequisites.wait_until_objects_created(
+                [(fip_addr, 'pwd')],
+                self.migration_utils.wait_until_vm_accessible_via_ssh,
+                TIMEOUT)
```
Comment (roman-verchikov) [inline cloudferry_devlab/cloudferry_devlab/generate_load.py]: I'd suggest that we have `TIMEOUT` value configurable in `config.py`

**Comment by SVilgelm** [comment]
Also ``VM_ACCESSIBILITY_ATTEMPTS`` variable is not needed anymore, delete it

File: cloudferry_devlab/cloudferry_devlab/generate_load.py (hunk 1 above)
Comment (plomakin) [inline cloudferry_devlab/cloudferry_devlab/generate_load.py]: sorry, didn't see that. will fix it.

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by plomakin** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by plomakin** [comment]
+1, can be merged

**Pull request merged**